<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎使用菜品推荐系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addbutton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>25</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>添加菜品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setbutton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>偏好设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="historybutton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>历史记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intrecbutton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>570</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;
</string>
    </property>
    <property name="text">
     <string>智能推荐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ranrecbutton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>随机推荐</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>家园</string>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>燕南</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>农园</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>470</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>学五</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>学一</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>470</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
    </property>
    <property name="text">
     <string>勺园</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
